src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -69,6 +69,11 @@ public function copy($originFile, $targetFile, $override = false)
                 throw new IOException(sprintf('Failed to copy "%s" to "%s".', $originFile, $targetFile), 0, null, $originFile);
             }
 
+            if (is_executable($originFile)) {
+                // User Executable | Group Executable | Other Executable
+                chmod($targetFile, fileperms($targetFile) | 0x0040 | 0x0008 | 0x0001);
```

Comment:
please use the octal notation rather than the hexadecimal one here. It is much more common for permissions